integration-tests/soap-test/src/main/java/co/elastic/apm/soap/HelloWorldService.java

```diff
@@ -0,0 +1,31 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package co.elastic.apm.soap;
+
+import javax.jws.WebMethod;
+import javax.jws.WebService;
+import javax.jws.soap.SOAPBinding;
+
+@SOAPBinding(style = SOAPBinding.Style.RPC)
+@WebService(targetNamespace = "http://www.jboss.org/jbossas/quickstarts/wshelloworld/HelloWorld")
```

Comment:
Maybe use `javax.xml.XMLConstants#NULL_NS_URI` instead, or anything else, not to have this jboss thing here and in the two other places?

Reply:
Does not work without a namespace unfortunately. Just used `elastic` now. Doesn't really matter that much anyway.

Reply:
I'm sure doesn't matter, just not have this show up in "jboss" searches because it is irrelevant